config/application.rb

```diff
@@ -2,19 +2,7 @@
 
 require_relative 'boot'
 
-require 'rails'
-# Pick the frameworks you want:
-require 'active_model/railtie'
-require 'active_job/railtie'
-require 'active_record/railtie'
-require 'active_storage/engine'
-require 'action_controller/railtie'
-require 'action_mailer/railtie'
-require 'action_mailbox/engine'
-require 'action_text/engine'
-require 'action_view/railtie'
-require 'action_cable/engine'
-# require "rails/test_unit/railtie"
+require 'rails/all'
```

Comment:
I think that, this change effectively started requiring `rails/test_unit/railtie`. I think you do not use it because you use RSpec instead of Test Unit.